crates/oxc_language_server/src/snapshots/fixtures_linter_tsgolint@no-floating-promises_index.ts.snap

```diff
@@ -0,0 +1,80 @@
+---
+source: crates/oxc_language_server/src/tester.rs
+input_file: crates/oxc_language_server/fixtures/linter/tsgolint/no-floating-promises/index.ts
+---
+code: "typescript-eslint(no-confusing-void-expression)"
+code_description.href: "None"
+message: "Returning a void expression from an arrow function shorthand is forbidden. Please add braces to the arrow function."
+range: Range { start: Position { line: 0, character: 50 }, end: Position { line: 0, character: 66 } }
+related_information[0].message: ""
+related_information[0].location.uri: "file://<variable>/fixtures/linter/tsgolint/no-floating-promises/index.ts"
+related_information[0].location.range: Range { start: Position { line: 0, character: 50 }, end: Position { line: 0, character: 66 } }
+severity: Some(Warning)
+source: Some("oxc")
+tags: None
+fixed: Single(FixedContent { message: None, code: "{  resolve(\"value\"); }", range: Range { start: Position { line: 0, character: 49 }, end: Position { line: 0, character: 66 } } })
```

Comment:
not great, but a good start: 
<img width="824" height="67" alt="Image" src="https://github.com/user-attachments/assets/3febcb24-c9d2-409e-8c28-58b103811e68" />